devtools/prod-envs/openff-benchmark-analysis.yaml

```
@@ -0,0 +1,37 @@
+name: openff-benchmark-analysis
+channels:
+  - conda-forge
+  - omnia
+  - defaults
+  - omnia/label/rc
+dependencies:
+  - python =3.7
+  - pip
+  - qcfractal =0.15.1
+  - qcengine =0.17.0
+
+  # Storage dependencies
+  - alembic
+  - psycopg2 >=2.7
+  - postgresql
+  - sqlalchemy >=1.3
+
+  # QCPortal dependencies
+  - double-conversion >=3.0.0
+  - h5py
+  - pandas
+  - plotly >=4.0.0
+  - pyarrow >=0.13.0
+  - tqdm
+
+  # MM calculations
+  - openforcefield =0.8.0
```

Let's bump this up to `0.8.2` to get the stereochemistry fixes
```suggestion
  - openforcefield =0.8.2
```